backend/db/mongo.go

```diff
@@ -39,6 +42,34 @@ func (c *Database) insertArticle(article Article) (string, error) {
 // 	return User{}, nil
 // }
 
+// fetch all Article from mongo
+func (c *Database) FindAllArticles() (int64, []Article, error) {
+	collection := c.database.Collection("articles")
+	cursor, err := collection.Find(context.TODO(), bson.D{})
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	defer cursor.Close(context.TODO())
+
+	count, err := collection.CountDocuments(context.TODO(), bson.D{})
+	articles := make([]Article, count)
+	//if err := cursor.All(context.TODO(), &articles); err != nil {
+	//	log.Fatal(err)
+	//}
+	for cursor.Next(context.TODO()) {
+		var article Article
+		if err := cursor.Decode(&article); err != nil {
+			log.Fatal(err)
+		}
+		articles = append(articles, article)
+		fmt.Println(article)
```

Comment:
Lets make sure we get rid of this print, we can use log instead if you would like the server to log out the articles!